src/components/FieldMultiSelect/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from 'react';
+
+import { useField } from '@formiz/core';
+import { GroupBase } from 'react-select';
+
+import { FieldSelectProps, FormGroup, Select } from '@/components';
+
+export interface FieldMultiSelectProps<
+  Option,
+  IsMulti extends boolean = true,
+  Group extends GroupBase<Option> = GroupBase<Option>
+> extends FieldSelectProps<Option, IsMulti, Group> {
+  isNotClearable?: boolean;
+}
+
+export const FieldMultiSelect = <
+  Option,
+  IsMulti extends boolean = true,
+  Group extends GroupBase<Option> = GroupBase<Option>
+>(
+  props: FieldMultiSelectProps<Option, IsMulti, Group>
+) => {
+  const {
+    errorMessage,
+    id,
+    isValid,
+    isSubmitted,
+    resetKey,
+    setValue,
+    value,
+    otherProps,
+  } = useField(props);
+  const { required } = props;
+  const {
+    children,
+    label,
+    options = [],
+    placeholder,
+    helper,
+    noOptionsMessage,
+    isDisabled,
+    isNotClearable,
+    isSearchable,
+    size,
+    selectProps = {},
+    ...rest
+  } = otherProps;
+  const [isTouched, setIsTouched] = useState(false);
+  const showError = !isValid && (isTouched || isSubmitted);
+
+  useEffect(() => {
+    setIsTouched(false);
+  }, [resetKey]);
+
+  const formGroupProps = {
+    errorMessage,
+    helper,
+    id,
+    isRequired: !!required,
+    label,
+    showError,
+    ...rest,
+  };
+
+  const handleChange = (values) => {
+    if (!values || !values.length) {
+      setValue(null);
+      return;
+    }
+    setValue(values?.map((value) => value.value));
+  };
+
+  return (
+    <FormGroup {...formGroupProps}>
+      <Select
+        id={id}
+        value={options?.filter((option) => value?.includes(option.value)) || []}
+        onBlur={() => setIsTouched(true)}
+        placeholder={placeholder}
+        onChange={handleChange}
+        options={options}
+        isDisabled={isDisabled}
+        isClearable={!isNotClearable}
+        isSearchable={isSearchable}
+        noOptionsMessage={() => noOptionsMessage || 'Aucune option'}
```

Comment:
Please add at least english and french translations for this part.

```suggestion
        noOptionsMessage={() => noOptionsMessage || t('somethinghere')}
```